h2d/Graphics.hx

```diff
@@ -537,6 +537,31 @@ class Graphics extends Drawable {
 		lineTo(cx, cy);
 		flush();
 	}
+	
+	public function drawRectanglePie( cx : Float, cy : Float, width : Float, height : Float, angleStart:Float, angleLength:Float, nsegments = 0 ) {
+		if(Math.abs(angleLength) >= Math.PI*2) {
+			return drawRect(cx-(width/2), cy-(height/2), width, height);
+		}
+		flush();
+		lineTo(cx, cy);
+		if( nsegments == 0 )
+			nsegments = Math.ceil(Math.abs(width * angleLength / 4));

```

Comment:
should maybe be Math.max(width,height) ?

Reply:
yeah absolutely 👍 forgot that.